source/edge-server/configure.txt

```diff
@@ -595,27 +595,92 @@ multiple Edge Server instances on a single host - for example, your local
 machine. You can do this using the ``edgectl`` profile functionality.
 
 When you create an Edge Server instance, ``edgectl`` automatically creates
-a default profile for you. You never have to specify the profile while there
-is only one profile.
+a default profile for you. The profile maps your commands to a specific Edge
+Server instance.
+
+When you create a second Edge Server instance on the same machine, ``edgectl``
+sets the new profile as the default profile. When you want to issue a command 
```

Comment:
```suggestion
sets the new profile as the default profile. You can change this by 
manually :ref:`setting a default profile <set-default-edgectl-profile>`. 

When you want to issue a command 
```

Reply:
Actually, no - the point I'm trying to make here is that if you want to specify that a command use some other profile, pass the `--profile` flag - otherwise, it uses the default. There's a section below about changing the default, but that's a different flag.